### PR TITLE
Hulks and xenomorphs can unbuckle faster from being cuffed and buckled

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
:cl:
 * balance: Hulks and xenomorphs can now break out of being buckled in 5 seconds instead of 120 seconds.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
If you get cuffed/bucked as a xeno or a hulk, you fully deserve what happened to you and to embarrassingly wait the 120 seconds.

What's next, people with an antag role are immune to disarms ?

**Comment by Sooose** [comment]
You can already break cuffs in five seconds, this is just making it so the same applies for being buckled. Considering you're not always hulked if you have the genetic power it's pretty easy to get stunned and then cuffed.

**Comment by ghost** [comment]
Having 120 second unbuckle with hulk is clearly an oversight given that it shortens uncuff so drastically.

**Comment by ShiftyRail** [comment]
Did you try not to get buckled
More seriously, it removes a possibility to restrain hulks, waiting for their power to run out, and force you to kill them instead.

I don't believe this is a good idea. Hulk is already strong enough ; if you managed to be cuffed and buckled despite being unstunnable, then clearly you deserve what happened there.

**Comment by ghost** [comment]
What do you mean by "deserve"?

**Comment by Sooose** [comment]
Very easy to end up buckled despite being a hulk, consider this scenario. You've got the genetics power for hulking and you're shot with a taser. You're already guaranteed to be buckle cuffed before you can do anything about it. 
- Hulking while you're stunned doesn't unstun you. 
- You can't break cuffs while you're being dragged around even if the timer is only 5 seconds.
- Buckling someone cuffed to a chair is an instant process.

I don't see the problem with having to detain hulks peacefully as sec because there is already a clause in space law that says lethal force is the way security should be dealing with hulks.

Having said that, I don't care whether this gets merged or not but breaking out of bucklecuff with hulk should be the same timer as uncuffing if it isn't merged.

**Comment by jknpj** [comment]
ignore secmain posts
hide secmain posts
report secmain posts

**Comment by Dacendeth** [comment]
You can always just cut a hulks foot off to prevent him from uncuffing himself.

**Comment by bathosbathtime** [comment]
What are the buckles made of that a hulk takes so long to break them while simultaneously a spessman can break them in the same amount of time

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -960,22 +960,37 @@ Thanks.
...
-									  self_drugged_message="<span class='warning'>You attempt to regain control of your legs (this will take a while).</span>")
-					spawn(0)
-						if(do_after(usr, usr, 1200))
-							if(!C.locked_to)
-								return
-							C.visible_message("<span class='danger'>[C] manages to unbuckle themself!</span>",\
-								"<span class='notice'>You successfully unbuckle yourself.</span>",\
+					C.delayNextAttack(50)
+					C.delayNextSpecial(50)
+					if(isalienadult(C) || (M_HULK in usr.mutations))
```
Comment (jknpj) [inline code/modules/mob/living/living.dm:965]: aliens have hulk, but this might be saner this way

Comment (boy2mantwicethefam) [reply code/modules/mob/living/living.dm:965]: Does that mean it's possible to assault xenos with ryetalin to be able to stun them?

**Pull request merged**